packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.css

```diff
@@ -17,3 +17,56 @@ li {
 .facet__value-checkbox {
   min-width: unset;
 }
+
+.facet__value_new-checkbox{
+  width: 20px;
+  height: 20px;
+  vertical-align: top;
+  border: solid 1px #cccccc;
+  border-radius: 4px;
```

Comment:
There is already some [CSS variables provided by SLDS](https://www.lightningdesignsystem.com/design-tokens/#category-radius) that you can use here instead of using values directly. Using the CSS variables helps us to stay consistant across all the quantic component.